### ISSUE TITLE
Have an admin interface to promote user as admin?

### ISSUE THREAD

**Issue by pierreozoux** [tosdr/tosdr2#47]
https://github.com/gterrono/houston

@michielbdejong what do you think?

**Comment by michielbdejong** [comment]
Makes sense to add a view to edit user details (specifically, `isAdmin`), but I think we can also do it the normal Meteor way.
